blockchain/mkalloc.go

```diff
@@ -18,6 +18,7 @@
 // This file is derived from core/mkalloc.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
+//go:build none
 // +build none
```

Comment:
Shouldn't we remove unnecessary comment?

Reply:
It seems that `mkalloc.go` is not used anywhere. We can consider removing the file, but not for this PR.